README.md

```diff
@@ -26,6 +26,7 @@ Keybinding                              | Description
 <kbd>Ctrl+K</kbd>, <kbd>Shift+Tab</kbd> | Previous view.
 <kbd>Ctlr+J</kbd>, <kbd>Tab</kbd>       | Next view.
 <kbd>Ctrl+H</kbd>, <kbd>Alt+H</kbd>     | Toggle history.
+<kbd>Ctrl+E</kbd>                       | Export as cURL command
```

Comment:
Yeah, as a vim user the real reason I chose it is to annoy emacs users :trollface: